samples/dect/dect_phy/dect_shell/src/dect/common/dect_phy_api_scheduler.c

```diff
@@ -401,6 +418,26 @@ void dect_phy_api_scheduler_list_item_beacon_tx_sched_config_update_by_phy_op_ha
 	k_mutex_unlock(&to_be_sheduled_list_mutex);
 }
 
+void dect_phy_api_scheduler_list_item_sched_config_frame_time_update_by_phy_op_handle(
```

Comment:
I don't see this function being used anywhere.

Reply:
Same here, stay tuned for the usage. I wanted to split my changes to several PRes.